pymc3/step_methods/metropolis.py

```diff
@@ -300,10 +300,13 @@ class BinaryGibbsMetropolis(ArrayStep):
     """A Metropolis-within-Gibbs step method optimized for binary variables"""
```

Comment:
could you add a reference to antithetical sampling in the docstring?  I guess strict backwards compatibility would suggest a default of `transit_p=1.0`, but "stationary distribution backwards compatibility (TM)" is ok with any value?